docs/components/tasklist/userguide/find-relevant-tasks.md

```diff
@@ -0,0 +1,87 @@
+---
+id: find-relevant-tasks
+title: Find relevant tasks
+description: "Tasklist is a ready-to-use application to work on user tasks."
+---
+
+When you open the **Tasks** page, you will see the list of all tasks. This is a starting point for managing your workload.
+
+![tasklist-all-tasks](img/task-filters/tasklist-all-tasks.jpg "All open tasks")
+
+## Apply a filter
+
+As you browse through the list, you can utilize filters to locate specific tasks and organize your workload.
+
+### Select a filter
+
+You can select one of the default filters or [create a new filter](#create-new-filter).
+
+Default filters:
+
+- All open tasks
+- Assigned to me
+- Unassigned
+- Completed tasks
+
+![tasklist-default-filters](img/task-filters/tasklist-default-filters.jpg "List of the default filters")
+
+## Create new filter
+
+To create a new filter, click on the option to apply a new filter. This action will open a filter dialog.
```

Comment:
I've cleaned up some of the UI references, but there are a few cases where we say "click the X option" -- instead, I would just say "Click **Enter UI element**."

Reply:
Similar to how you did in the example use case doc! 👍